_gsocproposals/2020/proposal_ROOTIO.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Pre-conditioners applied to ROOT compression algorithms
+layout: gsoc_proposal
+project: ROOT
+year: 2020
+organization:
+  - UNL
+  - MIR
+---
+
+## Description
+
+Pre-conditioning is an algorithm that rearranges typed, binary data to improve compression. More information can be found here: [1],[2]
+
+The idea of the project is that different filter or transformation schemes that could be applied to ROOT data, which are stored in "baskets". 
```

Comment:
Missing a predicate here. Maybe: "The idea of the project is to develop different filtering or transformation schemes, that could be applied to ROOT data, while stored in intermediate I/O buffers (baskets)." ?

Reply:
Thanks a lot! Done!